README.md

```diff
@@ -1,20 +1,69 @@
-This is the official website for GWOC
+<h1 align="center">
+  GLUG WINTER OF CODE
+</h1>
+<p align="center">
+    This is the official website for Glug Winter of Code
+</p>
+<p align="center">
+  <a href="https://github.com/glugmvit/gwoc/actions">
+    <img src="https://github.com/glugmvit/gwoc/workflows/Build/badge.svg" alt="build" />
+  </a>
+  <a href="https://gwoc.glugmvit.com/">
+    <img src="https://img.shields.io/website-up-down-green-red/https/gwoc.glugmvit.com/.svg" alt="website" />
+  </a>
+  <br>
+  <a href="CONTRIBUTING.md">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome" />
+  </a>
+  <a href="https://github.com/glugmvit/gwoc/issues">
+    <img src="https://img.shields.io/github/issues/glugmvit/gwoc.svg" alt="issues" />
+  </a>
+  <a href="https://prettier.io/">
+    <img src="https://img.shields.io/badge/code_style-prettier-ff69b4.svg" alt="code-style" />
+  </a>
+  <br>
+  <a href="https://twitter.com/intent/follow?screen_name=glugmvit">
+    <img src="https://img.shields.io/twitter/follow/glugmvit.svg?label=Follow%20@glugmvit" alt="Follow @glugmvit" />
+  </a>
+</p>
 
-[![build](https://github.com/glugmvit/gwoc/workflows/Build/badge.svg)](https://github.com/glugmvit/gwoc/actions)
-[![website](https://img.shields.io/website-up-down-green-red/https/gwoc.glugmvit.com/.svg)](https://gwoc.glugmvit.com/)
 
-## Content:
+<h1 align="center">
+  Contributing to GWOC Website
+</h1>
 
-### Home
-This is contains the social links of GLUG the event name
-### About
-Information about GLUG and what WOC is and how to get started with it
-### Register
-Registrations for mentors and participants
-### Schedule
-A brief timeline of events
-### FAQ
-Some common questions asked by participants/mentors
 
+We love contributions! We want to make contributing to this project as easy and friendly as possible, whether it's:
 
-#### Users can fork the repo and start contributing
+- Beginner PR requests
+- Questions regarding any problems
+- Submitting a fix
+- Proposing new features
+- Suggestions and Feedbacks
+
+## We Use [Github Flow](https://guides.github.com/introduction/flow/index.html), So All Code Changes Happen Through Pull Requests
+
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://guides.github.com/introduction/flow/index.html)). We actively welcome your pull requests:
+
+1. Fork the repo and create your branch from `master`.
+2. Run the app in developer mode
+    ```
+    npm start
+    ```
+    Open [http://localhost:3000](http://localhost:3000) to view it in the browser. The page will reload as you make edits.
+
+3. If you've added code that should be tested, add tests.
```

Comment:
Let's just remove the test part as of now, tests aren't really expected with new contribution atm.